README.md

```diff
@@ -51,3 +51,28 @@ scrape_configs:
         replacement: 127.0.0.1:9275 # Nrpe exporter.
 
 ```
+
+## SSL support
+
+Add query parameter `ssl=1` to use NRPE over SSL.
+
+NRPE requires the ADH ciphersuite which is not built by default in modern
+version of openssl. If the following command returns nothing then you will
+not be able to use it:
+
+```
+openssl ciphers -s -v ALL | grep ADH   # remove -s on older versions of openssl
+```
+
+The solution is to build the `nrpe_exporter` binary on an older server -
+Ubuntu 16.04 works. This will link against `libssl-1.0.0`, which still exists
```

Comment:
That's dynamically linked though, so will only work on the older server.

Reply:
I've tested it and it works: building dynamic executable under 16.04 and running under 18.04.  This is because 18.04 has a `libssl.so.1.0.0` library for backwards compatibility.

However I'm happy to remove mention of this and just say that it needs to be built statically.

Reply:
Having one way is easiest.